elements/rh-button/rh-button.css

```diff
@@ -0,0 +1,490 @@
+  --_danger-hover-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_danger-hover-background-color: #a30000; /* WARNING: non-token value */
+  --_danger-hover-border-color: transparent;
+
+  /* SECONDARY */
+  --_secondary-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-background-color: transparent;
+  --_secondary-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-border-width: var(--rh-border-width-sm, 1px);
+  --_secondary-active-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-active-background-color: transparent;
+  --_secondary-active-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-active-border-width: var(--rh-border-width-md, 2px);
+  --_secondary-focus-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-focus-background-color: transparent;
+  --_secondary-focus-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-focus-border-width: var(--rh-border-width-md, 2px);
+  --_secondary-hover-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-hover-background-color: transparent;
+  --_secondary-hover-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-hover-border-width: var(--rh-border-width-md, 2px);
+
+  /* TERTIARY */
+  --_tertiary-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-background-color: transparent;
+  --_tertiary-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-border-width: var(--rh-border-width-sm, 1px);
+  --_tertiary-active-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-active-background-color: transparent;
+  --_tertiary-active-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-active-border-width: var(--rh-border-width-md, 2px);
+  --_tertiary-focus-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-focus-background-color: transparent;
+  --_tertiary-focus-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-focus-border-width: var(--rh-border-width-md, 2px);
+  --_tertiary-hover-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-hover-background-color: transparent;
+  --_tertiary-hover-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-hover-border-width: var(--rh-border-width-md, 2px);
+
+  /* LINK */
+  --_link-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_link-background-color: transparent;
+  --_link-active-color: var(--rh-color-interactive-blue-darkest, #004080);
+  --_link-active-background-color: transparent;
+  --_link-focus-color: var(--rh-color-interactive-blue-darkest, #004080);
+  --_link-focus-background-color: transparent;
+  --_link-focus-outline-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_link-hover-color: var(--rh-color-interactive-blue-darkest, #004080);
```

Comment:
`--_link-hover-color`, `--_link-active-color`, and `--_link-focus-color` should be `--rh-color-blue-600` to increase contrast with default state.